sdks/python/apache_beam/runners/direct/direct_runner.py

```diff
@@ -329,8 +279,3 @@ def aggregated_values(self, aggregator_or_name):
 
   def metrics(self):
     return self._evaluation_context.metrics()
-
-
-class EagerRunner(DirectRunner):
```

Comment:
<!--thread_id:cc_164250769_t; commit:0f554fd51414381646c32b36f6f87a14adc0fad8; resolved:1-->
<!--section:context-quote-->
> **robertwb** wrote:
> This was primarily useful to explicitly construct it (probably for testing). We really don't want to be Eager for anything but the most basic runner. 
> 
> However, I'm find with dropping this if it's not useful.

<!--section:body-->
Acknowledged.  I removed this because there were no uses.  We can work on implementing it if it becomes necessary.